From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Thu, 5 Mar 2020 05:58:24 +0000
Subject: ANDROID: Kconfig.gki: Drop GKI_QCOM_CLKS_CONFIGS

With the recent changes to support clks as modules, as well as
efforts to get rpmh/rpmhpd/rpmpd working as a module the
GKI_QCOM_CLKS_CONFIGS entry is no longer needed.

So remove it!

Bug: 146449535
Change-Id: I364d57e8447a599a9ca838b6f5d9cd23d9a1353e
Signed-off-by: John Stultz <john.stultz@linaro.org>
[joneslee: Removed defconfig elements]
[joneslee: Removed init/Kconfig.gki elements]
Signed-off-by: Lee Jones <lee.jones@linaro.org>
---
 arch/arm64/configs/db845c_gki.fragment | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -64,6 +64,8 @@ CONFIG_INTERCONNECT_QCOM=y
 CONFIG_INTERCONNECT_QCOM_OSM_L3=m
 CONFIG_INTERCONNECT_QCOM_SDM845=m
 CONFIG_INCREMENTAL_FS=m
+CONFIG_QCOM_RPMH=m
+CONFIG_QCOM_RPMHPD=m
 CONFIG_WLAN_VENDOR_ATH=y
 CONFIG_ATH10K_AHB=y
 CONFIG_ATH10K=m
